samples/FunctionApp/HttpTriggerSimple/HttpTriggerSimple.cs

```diff
@@ -12,6 +12,7 @@ namespace FunctionApp
 {
     public static class HttpTriggerSimple
     {
+        [FixedDelayRetry(5, "00:00:10")]
```

Comment:
I thought http triggers does not have retry. So this sample might be confusing for the users.